src/config.rs

```diff
@@ -62,6 +62,7 @@ pub struct Config {
     pub syntax_dummy_theme: SyntaxTheme,
     pub syntax_set: SyntaxSet,
     pub syntax_theme: Option<SyntaxTheme>,
+    pub width: Option<String>,
```

Comment:
The `config` struct already has `available_terminal_width` and `decorations_width`. I feel like it should not need another field holding a width value.

Reply:
fixed here
313378a